presto-main/src/main/java/com/facebook/presto/execution/UseTask.java

```diff
@@ -68,16 +74,16 @@ private void checkCatalogAndSessionPresent(Use statement, Session session)
         }
     }
 
-    private void checkAndSetCatalog(Use statement, Metadata metadata, QueryStateMachine stateMachine, Session session)
+    private void checkAndSetCatalog(Use statement, Metadata metadata, QueryStateMachine stateMachine, Session session, AccessControl accessControl)
```

Comment:
nit: `accessControl` is unused, please remove

Reply:
removed `accessControl` from method